software/control/filterwheel.py

```diff
@@ -0,0 +1,68 @@
+import os
+import time
+
+from control._def import *
+
+class SquidFilterWheelWrapper:
+
+    def __init__(self, navigationController):
```

Comment:
@veerwang it should only use microcontroller, not navigationController.

Reply:
OK